src/proj_4D_api.c

```diff
@@ -480,9 +481,18 @@ Returns 1 on success, 0 on failure
         double *d = P->datum_params;
         size_t n = strlen (s);
 
+
         /* We ignore null helmert shifts (common in auto-translated resource files, e.g. epsg) */
         if (0==d[0] && 0==d[1] && 0==d[2] && 0==d[3] && 0==d[4] && 0==d[5] && 0==d[6])
+        {
```

Comment:
Since the Pandoras box of code style discussion already has been opened, please follow the few [rules](http://proj4.org/development/for_proj_contributors.html#coding-style) we do have :-)
(in the rest of this file starting curly braces are on the same line as `if (...)` etc.

Reply:
Style fixed